packages/rx_bloc_cli/mason_templates/bricks/lib_change_language/__brick__/lib/lib_change_language/bloc/change_language_bloc.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:rx_bloc/rx_bloc.dart';
+import 'package:rxdart/rxdart.dart';
+import 'package:widget_toolkit/language_picker.dart' hide ErrorModel;
+
+import '../../base/extensions/error_model_extensions.dart';
+import '../../base/models/errors/error_model.dart';
+import '../services/custom_language_service.dart';
+
+part 'change_language_bloc.rxb.g.dart';
+
+/// A contract class containing all events of the ChangeLanguageBloC.
+abstract class ChangeLanguageBlocEvents {
+  void setCurrentLanguage(LanguageModel model);
+}
+
+/// A contract class containing all states of the ChangeLanguageBloC.
+abstract class ChangeLanguageBlocStates {
+  /// The loading state
+  Stream<bool> get isLoading;
```

Comment:
Please remove the not used states

Reply:
I have removed the isLoadinga and error states